HelpSource/themes/solarizedLight.css

```diff
@@ -0,0 +1,42 @@
+:root {
+    /* editor colors */
+    --color-cm-background: #fdf6e3;
+    --color-cm-border: #e1e1e1;
+    --color-cm-evaluated: #abcfdb;
+    --color-cm-keyword: #dc322f;
+    --color-cm-built-in: #b58900;
+    --color-cm-number: #6c71c4;
+    --color-cm-symbol: #b58900;
+    --color-cm-class: #268bd2;
+    --color-cm-primitive: #2aa198;
+    --color-cm-char: #cb4b16;
+    --color-cm-env-var: #d33682;
+    --color-cm-comment: #586e75;
+    --color-cm-string: #93a1a1;
+    --color-cm-text: #657b83;
+    --color-cm-selection: var(--color-cm-border);
+
+    /* derived colors
+    in future use `color-mix` to calculate them which allows for fewer variables
+    but currently the qt browser does not support it, see
+    https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix
+    */
+    --color-bg: var(--color-cm-background);
+
+    --color-warning-bg: var(--color-cm-evaluated);
+    --color-warning-fg: #dbecf1;
```

Comment:
Here there is a mistake, I meant:
    --color-warning-bg: #dbecf1;
    --color-warning-fg: var(--color-cm-keyword);